packages/core/src/js/replay/CustomMask.tsx

```diff
@@ -0,0 +1,63 @@
+import { logger } from '@sentry/utils';
+import * as React from 'react';
+import type { HostComponent, ViewProps } from 'react-native';
+import { UIManager, View, requireNativeComponent } from 'react-native';
```

Comment:
NIT: Removing the unused import should fix the lint issue.

```suggestion
import { UIManager, View } from 'react-native';
```